test/integration/targets/aws_ses_identity_dkim/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+# defaults file for aws_ses_identity_dkim
+
+dkim_identity_domain: 'example.com'
+dkim_identity_email: 'address@example.com'
```

Comment:
These should be based on the resource_prefx to prevent parallel test runs interacting on the same identities. I'd suggest:

    dkim_identity_domain: "{{ resource_prefix }}-domain.example.com"
    dkim_identity_email: "address@{{ resource_prefix }}-email.example.com"

The use of resource_prefix will ensure that each test run is using fresh domains. This is often important as the CI build runs python2 / python3 runs in parallel so we don't want conflicting resources that would make the build unstable.

I suggest using a different sub-domain for the two identities because  there are interesting interactions between the domain and the email since we may just be enabling DKIM for a single email but DKIM is fundamentally domain based. So by using a different sub-domain for the two cases we remove any possibility that they interact.

Reply:
Yep, makes sense. Done